db/queries/cards.sql

```diff
@@ -106,12 +109,22 @@ WITH inserted_rows AS (
                 @review_logs::JSONB[],
                 array_fill('[]'::JSONB, array[array_length(@alphagrams, 1)])
             )
-        )
+        ),
+        sqlc.narg(deck_id)::BIGINT
     ON CONFLICT(user_id, lexicon_name, alphagram) DO NOTHING
     RETURNING 1
 )
 SELECT COUNT(*) FROM inserted_rows;
 
+-- name: MoveCards :one
+WITH moved_rows AS (
+    UPDATE wordvault_cards
+    SET deck_id = $3
+    WHERE user_id = $1 AND lexicon_name = $2 AND alphagram = ANY(@alphagrams::text[])
+    RETURNING 1
+)
+SELECT COUNT(*) from moved_rows;
```

Comment:
someone told me:

> if you omit returning it would already return the count.
> and it seems sqlc allows accessing that with :execrows (instead of :one)

Reply:
oh I actually think this was included in this PR by mistake -- should be a future one. Just removing from this branch

Reply:
(but will fix on the appropriate branch)